server/models/List.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require("mongoose");
+
+const listSchema = new mongoose.Schema({
+    name: {
+      type: String,
+      required: true
+    },
+    products: [{ 
+        type: Schema.ObjectId, 
```

Comment:
this gives me an error when I import, It should be `mongoose.Schema.ObjectId`, same for the userId